l10n/css.json

```
@@ -919,6 +919,12 @@
     "ja": "絶対長とパーセント値の組み合わせで与えられるオフセットと原点のキーワードを、各項目として構成されるリスト。",
     "ru": "Список, каждый элемент которого состоит из: смещения, данного комбинацией абсолютной длины и процентов плюс ключевое слово"
   },
+  "listEachItemConsistingOfNormalLengthPercentageOrNameLengthPercentage": {
+    "en-US": "A list, each item consisting of the keyword normal, a length percentage, or a timeline range name and a length percentage"
```

Same type of suggestion, feel free to disregard:

```suggestion
    "en-US": "A list where each item may be 'normal', a length percentage, or a timeline range name and a length percentage"
```